web/src/main/java/com/graphhopper/http/SpatialRuleLookupBuilder.java

```diff
@@ -54,30 +55,80 @@ public static SpatialRuleLookup build() {
     }
 
     public static SpatialRuleLookup build(BBox bounds, double resolution, boolean exact) {
-        SpatialRuleLookup spatialRuleLookup = new SpatialRuleLookupArray(bounds, resolution, exact);
         try {
             GHJson ghJson = new GHJsonBuilder().create();
             JsonFeatureCollection jsonFeatureCollection = ghJson.fromJson(new FileReader(new File(SpatialRuleLookupBuilder.class.getResource("countries.geo.json").getFile())), JsonFeatureCollection.class);
 
-            // TODO find outer border of all SpatialRules and create SpatialRuleLookupArray only for these bounds
-
+            // TODO Filter out polyons that don't intersect with the given BBox, will be implicitly done later anyway
             for (SpatialRule spatialRule : rules) {
                 for (JsonFeature jsonFeature : jsonFeatureCollection.getFeatures()) {
                     if (spatialRule.getCountryIsoA3Name().equals(jsonFeature.getProperty("ISO_A3"))) {
                         Geometry geometry = jsonFeature.getGeometry();
                         if (!geometry.isPolygon())
                             continue;
                         spatialRule.setBorders(geometry.asPolygon().getPolygons());
-                        spatialRuleLookup.addRule(spatialRule);
                         break;
                     }
                 }
             }
+
+            Double minLon = null;
```

Comment:
Here we could use the same approach like in PathWrapper.calcRouteBBox via
bbox = BBox.createInverse and then bbox.update(lat,lon)?

But not sure if it fits perfectly